<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Gui::Dialog::DlgSettingsAddonManager</class>
 <widget class="QWidget" name="Gui::Dialog::DlgSettingsAddonManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>569</width>
    <height>783</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Addon Manager Options</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="Gui::PrefCheckBox" name="guiprefcheckboxhideunlicensed">
     <property name="text">
      <string>Hide addons without a license</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <property name="prefEntry" stdset="0">
      <cstring>HideUnlicensed</cstring>
     </property>
     <property name="prefPath" stdset="0">
      <cstring>Addons</cstring>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Gui::PrefCheckBox" name="guiprefcheckboxhidenonfsf">
     <property name="text">
      <string>Hide addons with non-FSF free/libre license</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <property name="prefEntry" stdset="0">
      <cstring>HideNonFSFFreeLibre</cstring>
     </property>
     <property name="prefPath" stdset="0">
      <cstring>Addons</cstring>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Gui::PrefCheckBox" name="guiprefcheckboxnonosi">
     <property name="text">
      <string>Hide addons with non-OSI-approved license</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <property name="prefEntry" stdset="0">
      <cstring>HideNonOSIApproved</cstring>
     </property>
     <property name="prefPath" stdset="0">
      <cstring>Addons</cstring>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Custom repositories</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableView" name="customRepositoriesTableView">
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="sortingEnabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="addCustomRepositoryButton">
       <property name="text">
        <string notr="true">…</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="removeCustomRepositoryButton">
       <property name="text">
        <string notr="true">…</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="proxyGroupBox">
     <property name="toolTip">
      <string>Use a proxy server for access to addon data</string>
     </property>
     <property name="title">
      <string>Proxy addon manager traffic</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="proxyMainButtonLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QPushButton" name="systemProxyButton">
          <property name="toolTip">
           <string>Use the system's proxy settings</string>
          </property>
          <property name="text">
           <string>System</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <property name="autoExclusive">
           <bool>true</bool>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="customProxyButton">
          <property name="toolTip">
           <string>Use custom proxy settings</string>
          </property>
          <property name="text">
           <string>Custom</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="autoExclusive">
           <bool>true</bool>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="proxyHostSettingLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="proxyHostLabel">
          <property name="text">
           <string>Host</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="proxyPortLabel">
          <property name="text">
           <string>Port</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLineEdit" name="proxyHostLineEdit"/>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="proxyColonSeparatorLabel">
          <property name="text">
           <string>:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLineEdit" name="proxyPortLineEdit"/>
        </item>
        <item row="1" column="3">
         <widget class="QToolButton" name="proxyTestButton">
          <property name="toolTip">
           <string>Test these proxy settings</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="proxyStatusTestGroupBox">
        <property name="title">
         <string>Connection Test</string>
        </property>
        <layout class="QHBoxLayout" name="proxyStatusTestLayout">
         <item>
          <widget class="QLabel" name="proxyStatusTestOutputLabel">
           <property name="text">
            <string></string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutScore">
     <item>
      <widget class="QLabel" name="label_score">
       <property name="text">
        <string>Score source URL</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Gui::PrefLineEdit" name="guipreflineeditscoresourceurl">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>The URL for the addon score data (see Addon Manager wiki page for formatting and hosting details)</string>
       </property>
       <property name="prefEntry" stdset="0">
        <cstring>AddonsScoreURL</cstring>
       </property>
       <property name="prefPath" stdset="0">
        <cstring>Addons</cstring>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Gui::PrefCheckBox</class>
   <extends>QCheckBox</extends>
   <header>Gui/PrefWidgets.h</header>
  </customwidget>
  <customwidget>
   <class>Gui::PrefLineEdit</class>
   <extends>QLineEdit</extends>
   <header>Gui/PrefWidgets.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
